app/assets/javascripts/controllers/ConfirmDestructiveFailedQueueActionController.coffee

```diff
@@ -0,0 +1,9 @@
+controllers = angular.module("controllers")
+controllers.controller("ConfirmDestructiveFailedQueueActionController", [
+  "$scope", "$modalInstance",
+  ($scope ,  $modalInstance) ->
+
+    $scope.ok     = -> $modalInstance.close()
+    $scope.cancel = -> $modalInstance.dismiss('cancel')
```

Comment:
There's probably an opportunity to just create one controller, one view, etc. for all dialogs, but this is a reasonable compromise without doing too much stuff